framework/web/AssetManager.php

```diff
@@ -476,6 +476,10 @@ public function publish($path, $options = [])
             throw new InvalidArgumentException("The file or directory to be published does not exist: $path");
         }
 
+        if (!is_readable($path)) {
```

Comment:
we copy to `$path` i.e. need check `is_writeable()`

Reply:
But this `is_writeable()` check will be called later in `publishFile()` and `publishDirectory()` for the whole assets directory, this `$path` part will be used only as a part of this path. In the original issue was mentioned that problem also occurs on `filemtime()` call, which means that there is no read-access, isn't it?

Reply:
@manchenkoff you right, I messed up with target path